extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -729,15 +732,18 @@ export class ProjectsController {
 
 		const itemType = templates.get(itemTypeName);
 		const absolutePathToParent = path.join(project.projectFolderPath, relativePath);
-		let itemObjectName = await this.promptForNewObjectName(itemType, project, absolutePathToParent, constants.sqlFileExtension, options?.defaultName);
+		const isItemTypePublishProfile = itemTypeName === constants.publishProfileFriendlyName || itemTypeName === ItemType.publishProfile;
```

Comment:
when would `itemTypeName` be the friendly name rather than the `ItemType`?

Reply:
It would be in the scenario where "Add Item..." is used instead of "Add Publish Profile". The quick pick returns the friendlyName.